index.js

```diff
@@ -19,6 +19,161 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    const pastaAlfredo = {
+      title: "Pasta Alfredo",
+      level: "Easy Peasy",
+      ingredients: ["Pasta", "Cream", "Salt", "Pepper", "Cheese", "Flour"],
+      cuisine: "Italian",
+      dishType: "main_course",
+      image: "",
+      duration: 30,
+      creator: "Alfredo",
+      created: "",
+    };
+    await Recipe.create(pastaAlfredo);
+    await console.log(pastaAlfredo.title); 
+
+
+    //INSERT 
+    await Recipe.insertMany([
```

Comment:
You have above, a variable called `data` that imports the data from `"./data.js"` so, you should use that variable in this method.